rero_ils/modules/items/api_views.py

```diff
@@ -323,3 +324,58 @@ def item_availability(item_pid):
     return jsonify({
         'availability': item.available
     })
+
+
+@api_blueprint.route(
+    '/<item_pid>/can_request/<library_pid>/<patron_barcode>', methods=['GET'])
+@check_authentication
+@jsonify_error
+def can_request(item_pid, library_pid, patron_barcode):
+    """HTTP request to check if a librarian can request an item for a patron.
+
+    required_parameters: item_pid, library_pid, patron_barcode
+    """
+    return is_librarian_can_request_item_for_patron(
+        item_pid, library_pid, patron_barcode)
```

Comment:
Why don't using immediately `is_librarian_can_request_item_for_patron` or insert the code of this method into ?